pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+<!--Please fill in the **bold** fields, and tick all applicable boxes.-->
+
+Added **your URL here**.
+
+**Explain what your list is about and why it is awesome enough to be included here.**
+
+- [ ] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md).
+- [ ] This PR (and commit) have a descriptive title.
+- The list I added
+
+  - [ ] is not a duplicate,
+  - [ ] has been around for a few weeks,
+  - [ ] only has awesome items with descriptions,
+  - [ ] contains the awesome badge,
```

Comment:
linkify `awesome badge` to https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge